docs/mesos/RpmInstallCalico.md

```diff
@@ -0,0 +1,30 @@
+# Add Calico Networking To Mesos with an RPM
+This guide will walk you through adding Calico networking to your Mesos cluster using an RPM that can be built from the [calico-mesos repository](https://github.com/projectcalico/calico-mesos). You must follow these steps on *each Agent in your cluster*.
+
+*This guide assumes you already have [Mesos + Net-Modules installed](https://github.com/projectcalico/calico-docker/tree/master/docs/mesos#mesos--netmodules) on your Agent(s).* 
+
+# Build and Install the RPM
+The Calico-Mesos RPM can easily be built via the [calico-mesos repository](https://github.com/projectcalico/calico-mesos).
+
+    $ git clone https://github.com/projectcalico/calico-mesos.git
+    $ cd calico-mesos
+    $ make rpm
+    $ sudo yum install dist/calico-mesos.rpm
+    
+Alternatively, you can download the rpm directly from the [calico mesos-repository releases](https://github.com/projectcalico/calico-mesos/releases/latest).
```

Comment:
We should make the "download the RPM" the default, and the build it yourself an alternate course.